src/Slider/Slider.js

```diff
@@ -0,0 +1,263 @@
+// @flow
+
+import * as React from 'react';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { View } from 'react-native';
+import MultiSlider from 'react-native-multi-slider';
+
+import Marker from './Marker';
+import { Text } from '../Text';
+import { StyleSheet } from '../PlatformStyleSheet';
+
+const LABEL_LENGTH = 200;
+const TRACK_HEIGHT = 4;
+
+type Props = {|
```

Comment:
Maybe I'm just blind. But shouldn't there also be something like `onValuesChange` between props so I can handle value/s changes?

Reply:
Nope, you're right - I will expose tomorrow following props: `onValuesChange`, `onValuesChangeStart`, `onValuesChangeFinish`.